airflow/providers/elasticsearch/provider.yaml

```diff
@@ -72,3 +73,20 @@ connection-types:
 
 logging:
   - airflow.providers.elasticsearch.log.es_task_handler.ElasticsearchTaskHandler
+
```

Comment:
On top of also moving `[elasticsearch]` section (which I thin makes sense) it should be accompanied by adding the link to elastisearch provider configuration in the Airflow documentation: (following https://github.com/apache/airflow/pull/32777). 

When we have more of those, we might want to choose to do it automatically but for now we need to add it "manually"